index.html

```diff
@@ -10862,6 +10862,41 @@ <h2>A project for first-time contributions</h2>
             <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
             <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
 
+            <!-- ________ Abhijith's card START ________  -->
```

Comment:
Your code is correct but the placement is wrong. Please place your code right before the line
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```

I will be able to merge your PR once the above change is done.

Reply:
Please check now. I did the changes as you asked

Reply:
Excellent. Thank you!